bgpd/bgp_labelpool.c

```diff
@@ -1131,7 +1131,7 @@ static void show_bgp_nexthop_label_afi(struct vty *vty, afi_t afi,
 				ifindex2ifname(iter->nh->ifindex,
 					       iter->nh->vrf_id));
 		tbuf = time(NULL) - (monotime(NULL) - iter->last_update);
-		vty_out(vty, "  Last update: %s", ctime(&tbuf));
```

Comment:
I think we should change bgp_btoa.c too?  Just so we don't have any cut-n-paste incidents

Reply:
Sure, agree it's better not to have the examples out there...